kde-apps/kproperty/kproperty-9999.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit kde5
+
+DESCRIPTION="A property editing framework with editor widget similar to what is known from Qt Designer"
```

Comment:
Description looks to long to me. "A " can be removed and maybe "to what is known from" replaced with "like".